src/vivarium_public_health/treatment/intervention.py

```diff
@@ -0,0 +1,43 @@
+from typing import NamedTuple
+
+from vivarium_public_health.exposure import Exposure
+from vivarium_public_health.exposure.effect import ExposureEffect
+from vivarium_public_health.utilities import EntityString, TargetString
+
+
+class Intervention(Exposure):
+    """A component for modeling access to an intervention. The modeling and implementation of
+    this component is similar to that of a risk factor where the risk is a loack or not having
+    access to the intervention. Interventions should be configured with in the format of
+    "intervention.intervention_name".
+
```

Comment:
We should beef up these docstrings. They are more important than the ones for the ABC parent classes